lib/simulation/openfisca/mapping/index.js

```diff
@@ -40,14 +40,17 @@ var applyRessources = function(src, dest, schema, dateDeValeur) {
 
     var ressourcesByType = _.groupBy(src.ressources, 'type');
     _.forEach(schema, function(definition, destKey) {
-        var srcKey, fn;
+        var srcKeys, fn;
         if (_.isString(definition)) {
-            srcKey = definition;
+            srcKeys = [definition];
+        } else if(_.isArray(definition)) {
+            srcKeys = definition;
         } else {
-            srcKey = definition.src;
+            srcKeys = [definition.src];
             fn = definition.fn;
         }
 
+    var mapRessources = function(srcKey) {
```

Comment:
Your indentation is messy: it shows you're still within the `_.forEach(schema, function(definition, destKey) {` anonymous function when you declare `mapRessources`.
To properly decrease nesting, you have to declare the `mapRessources` function _outside_ from that scope ([example](https://github.com/sgmap/mes-aides-api/blob/abbatements_pensions_alimentaires_versees/lib/simulation/openfisca/mapping/index.js#L155-L162)). This way, you won't have coupling through captured variables, and more readable code with named functions  :)

That might be a bit tough at once, so if you prefer, I can do it and we can discuss it afterwards. Otherwise, better revert aec37c0 and bdd3317.
Let me know!